.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        node-version: [12, 14, 16]
+        node-version: [14, 16]
```

Comment:
Do we want to add `18`?

Reply:
Sure. I can add it

Reply:
So I checked this. nodejs18 is not supported in ubuntu-18 which is the OS env here. And if I upgrade ubuntu version then the [setup-mysql action](https://github.com/ankane/setup-mysql) fails as it doesn't support mysql5.7 in higher ubuntu versions.
So to add support for node v18, we will need to first support mysql v8 and then update the workflow to have a conditional run where, it should run on mysql v5.7 for node versions 14 & 16. And for v18, it should run on mysql v8. I am creating a separate issue for it and will do it there.
@raymondfeng I hope that works for this PR.

Reply:
I'm fine not to enable 18 for now.

Reply:
Then I am going ahead to merge it @raymondfeng @dhmlau